datatypes/src/collections/feature_collection.rs

```diff
@@ -1630,11 +1630,11 @@ mod tests {
         }
 
         fn time_interval_size(length: usize) -> usize {
+            let base = 64;
+
             if length == 0 {
-                return 0;
+                return base;
             }
-
-            let base = 64;
             let buffer = (((length - 1) / 4) + 1) * ((8 + 8) * 4);
 
             base + buffer
```

Comment:
Was this wrong?

Reply:
But do we have a test that checks the length of a datacollection?
This should offer the answer, right?

Reply:
this IS a test with a data collection or is it not? Regardless any collection with 0 features should be the same right? I mean if there are no features und no columns the `byte_size` method in feature_collection.rs just does `self.table.get_array_memory_size()` and if that now return 64 for an empty array I guess there is not much we can do right?